scripts/api-gateway-integration.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+
+"""
+Accepts WayFinder OpenAPI specification input from "standard in".
+Adds AWS API Gateway integrations to Lambda backend
+Output passed to "standard out"
+"""
+
+
+ServiceRequestIntegration = {
+    "type": "AWS_PROXY",
+    "httpMethod": "POST",
+    "uri": "${service_request_lambda_invoke_arn}",
```

Comment:
Where are these uri's being templated from? Should this script be loading some environment variables here and using a format string to add in these uris?

Reply:
They stay embedded in the final JSON file as they go up to S3. That JSON file is then read into terraform's template engine and replaces with the actual lambda ARNs on a per environment basis, before processing in API Gateway